repository-meta-analyzer/src/test/java/org/dependencytrack/repometaanalyzer/repositories/MavenMetaAnalyzerTest.java

```diff
@@ -63,7 +64,7 @@ void testAnalyzer() throws Exception {
         component.setPurl(new PackageURL("pkg:maven/junit/junit@4.12"));
         Assertions.assertEquals("MavenMetaAnalyzer", analyzer.getName());
         Assertions.assertTrue(analyzer.isApplicable(component));
-        Assertions.assertEquals(RepositoryType.MAVEN, analyzer.supportedRepositoryType());
+        Assert.assertEquals(RepositoryType.MAVEN, analyzer.supportedRepositoryType());
```

Comment:
for code quality: let's stay consistent and use 
```suggestion
        Assertions.assertEquals(RepositoryType.MAVEN, analyzer.supportedRepositoryType());
```